<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1295</width>
    <height>677</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Encrypted chat</string>
  </property>
  <property name="windowIcon">
   <iconset theme="QIcon::ThemeIcon::InsertLink"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="OutWindow">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>1271</width>
      <height>301</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTextEdit" name="sendEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>520</y>
      <width>631</width>
      <height>111</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="SendButton">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>540</y>
      <width>80</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
   </widget>
   <widget class="QListWidget" name="clientsListWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>161</width>
      <height>201</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="IPEdit">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>160</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>IP Address</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="PortEdit">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>190</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Port Number</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ConnectButton">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>180</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="nameEdit">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>130</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Your name</string>
    </property>
   </widget>
   <widget class="QPushButton" name="VoIPButton">
    <property name="geometry">
     <rect>
      <x>1170</x>
      <y>520</y>
      <width>111</width>
      <height>111</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Start a call</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset theme="QIcon::ThemeIcon::CallStart"/>
    </property>
   </widget>
   <widget class="QLabel" name="ActiveVoice">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>520</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="bAddFile">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>600</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset theme="QIcon::ThemeIcon::MailAttachment"/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1295</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
